wiremock/src/integTest/java/com/cognifide/aem/stubs/wiremock/AdminConsoleTest.java

```diff
@@ -0,0 +1,40 @@
+package com.cognifide.aem.stubs.wiremock;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+
+import org.junit.jupiter.api.Test;
+
+
+public class AdminConsoleTest {
+
+  @Test
+  public void shouldReturnAllScenarios() {
+    given()
+      .when()
+      .get("http://localhost:4502/stubs/__admin/scenarios")
+      .then()
+      .body("scenarios", hasSize(1))
+      .statusCode(200);
+  }
+
+  @Test
+  public void shouldResetScenarios() {
+    given()
+      .when()
+      .post("http://localhost:4502/stubs/__admin/scenarios/reset")
+      .then()
+      .statusCode(200);
+  }
+
+  @Test
+  public void shouldReturnMappingForScenario() {
```

Comment:
It would be nice to have a test scenario that really makes a few calls and test if responses change.

1. start with a reset
2. execute call - remember body value
3. execute call - assert current body is different from previous
4. reset again
5. execute call - assert current body is the same as in the firs call

Reply:
ok. I added such test

Reply:
great :)